cvat-ui/src/components/cloud-storages-page/empty-cloud-storages-list.tsx

```diff
@@ -45,7 +45,7 @@ export default function EmptyStoragesListComponent(props: Props): JSX.Element {
 
     return (
         <div className='cvat-empty-cloud-storages-list'>
-            <Empty description={description} image={<CloudTwoTone className='cvat-empty-cloud-storages-list-icon' />} />
+            <Empty description={description} image={<CloudOutlined className='cvat-empty-cloud-storages-list-icon' />} />
```

Comment:
Probably can be a secondary color not black

Reply:
I would agree here.

Reply:
![image](https://user-images.githubusercontent.com/40690378/152490458-84409e44-8888-4d3f-807f-fe99a6dfe988.png)